tests/unit/YourCompany.spec.ts

```diff
@@ -103,8 +104,8 @@ describe('YourCompany in a SP alteration', () => {
     store.state.stateModel.tombstone.entityType = entitySnapshot.businessInfo.legalType
     store.state.stateModel.entitySnapshot = entitySnapshot
     store.state.stateModel.tombstone.filingType = 'alteration'
-    store.state.stateModel.tombstone.entityType = 'SP'
-    store.state.resourceModel = BenefitCompanyResource
```

Comment:
This was the wrong entity type. (SPs don't have alterations; they have change of registrations.)